### PR TITLE
Support subset

### PR DESCRIPTION
**Pull request by shuheiktgw**
Add subset support using the Launchable CLI.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
+launchable>=1.1.3
```
Comment (ninjinkun) [inline requirements.txt:1]: The CLI may change its interface in the future. You have to fix the version to 1.x with `launchable >= 1.1.3, ~= 1.0`.

Comment (shuheiktgw) [reply requirements.txt:1]: Ah, I didn't know I can use both `>=` and `=~`. Thank you!

Comment (shuheiktgw) [reply requirements.txt]: @ninjinkun Fixed it so would you review the PR again?

**Review by ninjinkun — APPROVED** [review]
LGTM!

**Comment by shuheiktgw** [comment]
Thank you!

**Pull request merged**